app/views/account/pages/_index.html.erb

```diff
@@ -22,14 +22,15 @@
           <table class="table">
             <thead>
               <tr>
+                <th class="w-6"></th>
                 <%= render "shared/tables/select_all" %>
                 <th><%= t('.fields.name.heading') %></th>
                 <%# 🚅 super scaffolding will insert new field headers above this line. %>
                 <th><%= t('.fields.created_at.heading') %></th>
                 <th class="text-right"></th>
               </tr>
             </thead>
-            <tbody data-controller="sortable" data-sortable-reorder-path-value="<%= url_for [:reorder, :account, context, collection] %>">
+            <tbody data-controller="new-sortable" data-new-sortable-reorder-path-value="<%= url_for [:reorder, :account, context, collection] %>">
```

Comment:
`new-sortable` isn't a great name. If we can be backwards compatible enough I'd love to move the old `dragula` based controller to `dragula-sortable` or something and make the new version `sortable`. That would make for the most seamless upgrade for people.